src/sst/core/impl/timevortex/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# SST-CORE src/sst/core/impl/timevortex CMake
+#
+# Copyright 2009-2021 National Technology and Engineering Solutions of Sandia,
+# LLC (NTESS).  Under the terms of Contract DE-NA-0003525, the U.S. Government
+# retains certain rights in this software.
+#
+# See LICENSE for full license details
+#
+
+add_library(timeVortex OBJECT timeVortexPQ.cc)
+
+target_include_directories(timeVortex PUBLIC ${sst-core_SOURCE_DIR}/../src)
```

Comment:
These .. in the paths are a bit messy. I would define variables in the top-level cmake to make it clear what these actually are. You have a SST_CORE_TOP_CMAKE_DIR and SST_CORE_TOP_SOURCE_DIR.

Reply:
This will also make it easier if you ever move the cmake out of experimental.

Reply:
@jleidel we should do this.

Reply:
@feldergast this is an `OBJECT` library so it's just a collection of object files with some metadata (like the fact they depend on certain includes).  When you call `target_link_libraries(exe timeVortex)` against this it will look like 
```
c++ <flags> -o exe main.o timeVortextPQ.o
```
does that make sense? No actual library ever gets generated.

Reply:
Sounds good.

Reply:
@jleidel I'll mark this as resolved and we don't need to do it for this pr.